src/View/AdminUser/AddUserUI.java

```diff
@@ -0,0 +1,151 @@
+
+        leftPanel = makeLeftPanel();
+        rightPanel = makeRightPanel();
+        acceptBtn = new JButton("Accept");
+        cancelBtn = new JButton("Cancel");
+        this.setLayout(new BorderLayout());
+
+        acceptBtn.addActionListener(new UserEvent(this));
+
+        JPanel bottomPanel = new JPanel();
+        bottomPanel.setLayout(new BoxLayout(bottomPanel, BoxLayout.X_AXIS));
+
+        bottomPanel.add(acceptBtn);
+        bottomPanel.add(cancelBtn);
+
+        this.add(leftPanel, BorderLayout.LINE_START);
+        this.add(rightPanel, BorderLayout.LINE_END);
+        this.add(bottomPanel, BorderLayout.PAGE_END);
+
+        this.setBorder(BorderFactory.createTitledBorder("Add New User"));
+    }
+    /**
+     * Method to build the left panel with the corresponding data
+     * @return return a panel with all their components
+     */
+    private JPanel makeLeftPanel(){
+        JPanel panel = new JPanel(new SpringLayout());
+        Dimension txtSize = new Dimension(150, 25);
+        InitComponent.initLabel(nameLabel, "Name", panel);
+        InitComponent.initTextBox(nameTxt, txtSize, panel);
+
+        InitComponent.initLabel(lastNameLabel, "Last Name", panel);
+        InitComponent.initTextBox(lastNameTxt, txtSize, panel);
+
+        InitComponent.initLabel(userLabel, "User", panel);
+        InitComponent.initTextBox(userTxt, txtSize, panel);
+
+        InitComponent.initLabel(passwordLabel, "Password", panel);
+        InitComponent.initTextBox(passwordTxt, txtSize, panel);
+        InitComponent.initLabel(activeLabel, "Active", panel);
+
+        activeCbox = new JCheckBox();
+        panel.add(activeCbox);
+
+        InitComponent.setSizeComponent(panel, new Dimension(310, 50));
+        SpringUtilities.makeCompactGrid(panel, 5, 2, 6, 6, 6, 6); /* rows, cols, initX, initY, xPad, yPad*/
+
+        return panel;
+    }
+    /**
+     * Method to build the right panel with the corresponding data
+     * @return return a panel with all their components
+     */
+    private JPanel makeRightPanel()
+    {
+        JPanel rightPanel = new JPanel(new SpringLayout());
+        Dimension txtSize = new Dimension(150, 25);
+        InitComponent.initLabel(addressLabel, "Address", rightPanel);
+        InitComponent.initTextBox(addressTxt, txtSize, rightPanel);
+
+        InitComponent.initLabel(emailLabel, "E-mail", rightPanel);
+        InitComponent.initTextBox(emailTxt, txtSize, rightPanel);
+
+        InitComponent.initLabel(possitionLabel, "Possition", rightPanel);
+        InitComponent.initComboBox(possitionCbox, txtSize, rightPanel);
+
+        InitComponent.initLabel(roleLabel, "Role", rightPanel);
+        InitComponent.initComboBox(roleCbox, txtSize, rightPanel);
+
+        InitComponent.setSizeComponent(rightPanel, new Dimension(310, 50));
+        SpringUtilities.makeCompactGrid(rightPanel,4,2,6,6,6,6); /* rows, cols, initX, initY, xPad, yPad*/
+
+        return rightPanel;
+    }
+
+    public String getName(){
```

Comment:
you dont need getters to access component attributes unless you want to process the data before to use it